### ISSUE TITLE
New Themes

### ISSUE THREAD

**Issue by NeuroCPP** [Seneral/FlagPlayer#21]

**Comment by Seneral** [comment]
Sounds good!

**Comment by NeuroCPP** [comment]
it will take some time as i am busy with studies..but will try to complete it

**Comment by NeuroCPP** [comment]
only have to mess around with css properties files right? 
let's see I should be able to handle the html and adding the new drop menu for the new theme , can't promise it though
have a exam too 😬

**Comment by Seneral** [comment]
Very easy to add new themes, look how dark/light is handled. Hard part is making good themes lol

Think all you need to do is add the html line to the settings next to light and dark (with value "DARKGREEN" or something), and then you can customise css with theme-darkgreen, e.g. the main colors at the top of the css file. Will also need to change dark-icon behaviour if you want the dark icon in your theme and not the light.

**Comment by NeuroCPP** [comment]
> Hard part is making good themes lol

that's true fo' sure i was changing --acc-color value and it changed the player colors and flagplayer logo so i was keeping it blue for a while ..it is good on dark tho

**Comment by NeuroCPP** [comment]
new theme is done ..its light one see how it looks

**Comment by NeuroCPP** [comment]
I'll attach the changes made from my fork

**Comment by NeuroCPP** [comment]
> new theme is done ..its light one see how it looks



> new theme is done ..its light one see how it looks

**Comment by NeuroCPP** [comment]
> new theme is done ..its light one see how it looks

lost the changes as I force fetched the changes from my fork to local branch 😭(this reverted all the changes ... hard lesson learned)

**Comment by NeuroCPP** [comment]
re-did it 
https://github.com/Seneral/FlagPlayer/commit/92b81359dc14816147ca1fef2df2af04edf00cb2
here's the commit
NOTE: little bit of remaining like search bar and search button and stuff like that but it's little hard for that..
did it before .. I will checkout if it matches the theme this time
PS: looks better than first try and blue one at that

**Comment by Seneral** [comment]
Can you integrate it properly?
As I described it's just two lines, you add a new css block:
`body.theme-blue { /* Dark Blue Skin */`
and a new Theme dropdown element:
`<option value="BLUE">Blue</option>`
And that's it, it's now selectable as the dropdown.
The icon is also not important, it will always have a white pole (without modifying css further as I said) but other than that you can control the color with the acc-color.
So super easy to add as a new theme.
That said, yeah, needs some polishing:
- Playlist icon on playlist elements is nearly invisible
- Error messages (e.g. no internet) are very hard to read
- Primary and secondary texts (as well as borders) having opposing colors (blue vs red-brownish) looks weird
- Link color (acc-sec-color) looks weird blue on blue

But main colors are good, easy on the eyes

**Comment by Seneral** [comment]
Makes me want to play around with it as well, notice how I wanted a purple or blue theme from the start, saved some colors below.

**Comment by NeuroCPP** [comment]
> Makes me want to play around with it as well, notice how I wanted a purple or blue theme from the start, saved some colors below.

yeah I noticed it purple but never thought they were trials of new themes 





> Can you integrate it properly? As I described it's just two lines, you add a new css block: `body.theme-blue { /* Dark Blue Skin */` and a new Theme dropdown element: `<option value="BLUE">Blue</option>` And that's it, it's now selectable as the dropdown. The icon is also not important, it will always have a white pole (without modifying css further as I said) but other than that you can control the color with the acc-color. So super easy to add as a new theme. That said, yeah, needs some polishing:
> 
> * Playlist icon on playlist elements is nearly invisible
> * Error messages (e.g. no internet) are very hard to read
> * Primary and secondary texts (as well as borders) having opposing colors (blue vs red-brownish) looks weird
> * Link color (acc-sec-color) looks weird blue on blue
> 
> But main colors are good, easy on the eyes

1.yeah i will find an alternative for that or should it be left as it is
2.thanks for that I am learning on how to make a theme out of scratch now..it's not as thought 😅
3.about border colors I also found it not very good and very distracting too
4.oh I did not touch that making others were a real headache for me 😆

**Comment by NeuroCPP** [comment]
well i have to go off because of exams again hm don't know when will be back .. i hope someone else interested will help out too
😄
